src/core_ocean/shared/mpas_ocn_tidal_potential_forcing.F

```diff
@@ -0,0 +1,663 @@
+! Copyright (c) 2013,  Los Alamos National Security, LLC (LANS)
+! and the University Corporation for Atmospheric Research (UCAR).
+!
+! Unless noted otherwise source code is licensed under the BSD license.
+! Additional copyright and license information can be found in the LICENSE file
+! distributed with this code, or at http://mpas-dev.github.com/license.html
+!
+!|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
+!
+!  ocn_tidal_potential_forcing
+!
+!> \brief MPAS ocean tidal potential forcing module
+!> \author Steven Brus 
+!> \date   September 2019
+!> \details
+!>  This module contains routines for the tidal potential forcing.
```

Comment:
@sbrus89, can we please provide a **read-only** link to the design document here?

Reply:
@pwolfram, Do I have to have some kind of permissions to do that? I don't see where a can get a link under the share tab.

Reply:
@pwolfram and @sbrus89, I think the right way to do this is to put a pdf (and maybe a tex file) of the design document here: 
https://github.com/MPAS-Dev/MPAS-Model/tree/ocean/develop/docs/ocean/design_docs

I don't believe Overleaf an appropriate space to try to share design docs from -- it is only an authoring tool, not meant for read-only sharing.  Projects typically get archived when they are no longer used.

Reply:
@xylar, that's a good suggestion.

@pwolfram , I went ahead and added pdf and tex files of the design doc to the location Xylar pointed out. I also made a note in the header.

Reply:
@sbrus89, I'm going to move the overleaf to the repo @xylar recommended.  We just want to link there or have some comment about where to find the design doc.  Ideally, this would include the PR number for the design_doc.  Please feel free to begin squashing and we can just update that reference at the end.  I probably need 20min to get the design doc migrated over, so working in parallel is probably good...

Reply:
If I can push to your fork this could be easy and you could just rebase from your remote or cherry-pick the commit and then squash it with the rest.

Reply:
Looks like you already grabbed it, good work!

Reply:
Please feel free to squash down and we can move forward.

Reply:
I made a note on the overleaf of 

MPAS-O Ocean Tides Design Document Snapshot on Oct 18, 2019 placed at https://github.com/MPAS-Dev/MPAS-Model/tree/ocean/develop/docs/ocean/designdocs

So we know we moved it over to the MPAS today.